test/functional/feature_block.py

```diff
@@ -698,6 +700,13 @@ def run_test(self):
         self.send_blocks([b55], True)
         self.save_spendable_output()
 
+        # The block which was previously rejected because of being "too far(3 hours)" must be accepted 2 hours later.
+        # The new block is only 1 hour into future now
+        self.log.info("Accept a previously rejected future block at a later time")
+        node.setmocktime(int(time.time()) + 2*60*60)
+        self.send_blocks([b48], success=False) # The tip is not updated because b55 is seen first
```

Comment:
Not sure how useful it is to test with success=False

Maybe add a block and invalidate it after success?

Reply:
`success=False` would still error if given an invalid block. It only marks whether the tip was updated. 
It is however still a good addition to the test to actually show the reorg. Updated the test